FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m

```diff
@@ -327,20 +327,22 @@ - (void)URLSession:(NSURLSession *)session
 }
 
 - (void)receiveWebSocketData {
+    __weak FWebSocketConnection *weakSelf = self;
```

Comment:
I find `__weak __autotype weakSelf = self` as it is more compact and is better handled by Xcode refactoring (no need to be changed), but it's definitely up to you.